bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -90,21 +90,35 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{
           {% endtrans %}
           </p>
 
           <p>
           {% trans %}
-            Mitchell serves as a Board Member for the OpenMRS, which builds the world’s leading open source enterprise electronic medical record system platform. She is an MIT Media Lab Research Affiliate, Open Agriculture Initiative, co-chairs the Department of Commerce Digital Economy Advisory Board and serves on the United Nations High Level Panel on Women’s Economic Empowerment. Mitchell was co-Chair of the World Economic Forum Annual Meeting of the New Champions in 2015.
+            Mitchell is a Board Member for the OpenMRS, the world’s leading open source enterprise electronic medical record system platform, and is an MIT Media Lab Research Affiliate, Open Agriculture Initiative. She co-chairs the U.S. Department of Commerce Digital Economy Advisory Board and serves on the United Nations High Level Panel on Women’s Economic Empowerment. Mitchell was co-Chair of the World Economic Forum Annual Meeting of the New Champions in 2015 and a member of the ICANN High Level Panel on Global Internet Cooperation and Governance Mechanisms.
           {% endtrans %}
           </p>
+
+          <p>
+          {% trans %}
+            A native of the San Francisco Bay Area, Mitchell received her B.A. in Asian Studies from UC Berkeley and her J.D. from Berkeley Law. In her early career, she worked as a Corporate and Intellectual Property Lawyer at Fenwick West LLP and as Associate General Counsel for Sun Microsystems.
+          {% endtrans %}
+          </p>
+
+          <p>
+            <em>
+            {% trans %}
+              TIME Magazine profiled Mitchell in its global list of “100 Most Influential People” under “Scientists and Thinkers”. Bloomberg listed her as one of the “25 Most Influential People on the Web”. She was honored as winner of the Anita Borg Institute’s 2009 Women of Vision Award and received the Aenne Burda Award for Creative Leadership in 2010. In 2012, Mitchell was inducted into the founding group of the <a rel="external" href="http://www.internethalloffame.org/inductees/mitchell-baker">Internet Society’s Hall of Fame</a>. She has appeared on NBC “Meet the Press”, BBC “HardTalk”, “The Charlie Rose Show” and “CNN Global Office”. She has spoken at high-level events like the World Economic Forum, Mobile World Congress, The Web Summit, Collision, Singularity University and Bits&Pretzels.
```

Comment:
Nit: the `href` here should probably not be part of the string and passed in as a variable instead.